plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -135,4 +141,22 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
       }
     }
   }
+
+  // TODO obtain libraries from extension registry/osgi service
+  // https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/819
+  private boolean validateDependencies() {
+    try {
+      return
+          (MavenUtils.isArtifactAvailableLocally("javax.servlet", "servlet-api", "2.5")
+              || MavenUtils.resolveArtifact(new NullProgressMonitor(),
+                                            "javax.servlet", "servlet-api", "jar", "2.5") != null)
+          && (MavenUtils.isArtifactAvailableLocally("javax.servlet.jsp", "jsp-api", "2.1")
+              || MavenUtils.resolveArtifact(new NullProgressMonitor(),
+                                            "javax.servlet.jsp", "jsp-api", "jar", "2.1") != null);
```

Comment:
Without network connection on my laptop it took about 2 seconds to get the error message. Obviously if the connection is just very slow, then it may take a long time as we won't get an exception.